### ISSUE TITLE
Make /bin the main exec dir(Pull Request, (This title used to include a rant with crass language)

### ISSUE THREAD

**Issue by YusufKhan-gamedev** [Andy-Python-Programmer/aero#29]
https://github.com/YusufKhan-gamedev/uwux/commit/602904b5a602985db608753b4bd1364cc3895612

HERE! I spent an hour finding how to pr in github and here it is! Also, maybe create gitlab mirror? Sorry for my crass language im extremely angry.

**Comment by Andy-Python-Programmer** [comment]
You should refrain from using such language in issues. This could result in a ban if you don't stop doing it. It's really not required. Issues are ment to be friendly discussions about an issue.

> spent an hour finding how to pr in github and here it is!

I believe you should level up your searching skills and doing a simple pull request is not hard at all. Also your git tree is filled up with nonsense commits so please don't.

> Also, maybe create gitlab mirror? 
I don't like gitlab it really does not have the benefits its of  GitHub. Could make a mirror in the future.

Also the commit you made is makes exec not really do what it's ment to do. It's ment to take the full path mate. Think if you compile a program in the root and try it exec it with this patch applied it would fail. It's very simple. Just think about it.

**Comment by YusufKhan-gamedev** [comment]
I deeply apologize, over the past 2 months many things regarding the shell on my system would mysteriously not work or bug out. That made me get angry a lot, sorry....

The command I used to initiate a pull request was not at fault after upgrading my packages the format I used worked on a dummy repository. 

> Also the commit you made is makes exec not really do what it's ment to do. It's ment to take the full path mate. Think if you compile a program in the root and try it exec it with this patch applied it would fail. It's very simple. Just think about it.

Now I realize your intention.... shouldn't you add `./program` support though? Under most unix-like Operating Systems `program` looks at `/usr/bin` or `/usr/local/bin` and executes the program found there while `./program` executes a program in the directory its placed in, and `exec program` quits the current process in favor of "program".
